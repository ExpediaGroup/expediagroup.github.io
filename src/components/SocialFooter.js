import React from 'react';
import styles from './SocialFooter.module.css';
import ArrowLink from "./ArrowLink";

function SocialFooter({text, link}) {
    return (
        <div className={styles.socialFooter}>
            <p>{text}</p>
            <ArrowLink link={link}/>
        </div>
    );
}

export default SocialFooter;
