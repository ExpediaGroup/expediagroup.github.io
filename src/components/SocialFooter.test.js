import React from 'react';
import renderer from 'react-test-renderer';
import SocialFooter from './SocialFooter';

jest.mock('./ArrowLink', () => 'ArrowLink');

const LINK = 'http://test-link';
const TEXT = 'test text';

it('renders correctly', () => {
    const tree = renderer
        .create(<SocialFooter link={LINK} text={TEXT}/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
