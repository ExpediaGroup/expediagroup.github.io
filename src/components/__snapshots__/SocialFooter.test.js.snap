// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="socialFooter"
>
  <p>
    test text
  </p>
  <ArrowLink
    link="http://test-link"
  />
</div>
`;
